src/lexedata/exporter/cognates.py

```diff
@@ -188,7 +189,8 @@ def create_excel(
                     # Transfer the cognateset comment to the first Excel cell.
                     if c_comment and col == 1 and cogset.get(c_comment):
                         cell.comment = op.comments.Comment(
-                            cogset["Comment"], __package__
+                            re.sub(f"-?{__package__}", "", cogset["Comment"]).strip(),
+                            "lexedata.exporter",
```

Comment:
Why do you add the package name as the author of the comment?

Reply:
There must be an author for every comment. How could we derive a sensible author if not from either `__package__` (`"lexedata.exporter"`) or `"lexedata"`?